gameoflife.php

```diff
@@ -27,21 +46,62 @@ function autoload ($className)
  */
 $life = new Board(10, 10);
 
+if ($options->getOption("startRandom"))
+{
+    $life->generateRandomBoard();
+}
+
+if ($options->getOption("startGlider"))
+{
+    $life->generateGleiter();
+}
+
+if ($options->getOption("width"))
+{
+    echo $life->width;
+}
+
+if ($options->getOption("height"))
+{
+    echo $life->height;
+}
```

Comment:
That does not make sense! What use is it to output the height of the board?
These options are there to set the width and height of the board. So you must pass the value of this option to the constructor of your Board-class. Currently you hardcoded that to 10,10, but that should only be the default values if no height or width is passed as parameter. If they are passed as parameter, the board must be the size specified!